### PR TITLE
Create FAQs .md

### PR DESCRIPTION
**Pull request by omideleayo**
This is the document for NoteBuddy FAQs section

### CODE REVIEW

File: FAQs .md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# NoteBuddy FAQs
+
+
+Welcome to the FAQs section of NoteBuddy.
+This document is available to assist new and existing users of NoteBuddy to navigate easily on their preferred note taking App.
```
Comment (YoeriNijs) [inline FAQs .md]: This section is obsolete; we assume NoteBuddy is the preferred note taking app already ;)

File: FAQs .md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Why the name NoteBuddy?
+NoteBuddy sounds cool and shows having a friend to help create notes, store, and save notes for future reference.
+
+#### How do I start making notes on NoteBuddy?
+
+Once you are logged in, click on the pencil icon at the bottom right of the screen, input the preferred title of your note, (this help users scroll through the note list).
+Type or paste words in the **Note** box.
+You can paste notes from anywhere on your device into your NoteBuddynotes.
```
Comment (YoeriNijs) [inline FAQs .md]: NoteBuddy notes

File: FAQs .md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Select the save icon at the bottom right this automatically saves the notes.
+
+#### How do I backup my notes?
+
+Click on the menu icon at the top left of the screen and select **create backup**. 
+Then select either **decrypted** or **encrypted**, and a backup file would be generated depending on your selection.
+
+NoteBuddy uses what sort of encryption algorithm?
```
Comment (YoeriNijs) [inline FAQs .md]: 'Which encryption algorithm does NoteBuddy use?' sounds better to me. Furthermore, you can make this bold ;)

File: FAQs .md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+NoteBuddy uses the Advanced Encryption Standard (AES)algorithm implemented by Tozny'sfor encryption and decryption of data.
```
Comment (YoeriNijs) [inline FAQs .md]: Missing whitespaces: 
- (AES) algorithm
- Tozny's for

By the way, it is nicer if you create a link in Tozny's. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

File: FAQs .md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+https://github.com/tozny/java-aes-crypto
+
+#### What is Advanced Encryption Standard?
+
+Advanced Encryption Standard (AES) is a **symmetric-key algorithm** which implies the use of the same key for both encrypting and decrypting data.
+NoteBuddy works on all Android versions?
```
Comment (YoeriNijs) [inline FAQs .md]: On which Android versions does NoteBuddy work?

File: FAQs .md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+No, NoteBuddy is a note App available for devices running on Android 6.0 and above.
+
+#### What permission does NoteBuddyrequire?
+
+NoteBuddy only requires access to your device storage.
```
Comment (YoeriNijs) [inline FAQs .md]: You can mention read and write access here.

File: FAQs .md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Where is my NoteBuddy login details stored?
+
+For now, users login details are stored in the device Android Shared Preference.
```
Comment (YoeriNijs) [inline FAQs .md]: This sentence sounds a bit odd. What about: 'For now, login details are stored in Android's shared preferences.

File: FAQs .md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+#### What mixtures of characters are viable for use as NoteBuddy password?
+
+NoteBuddy allows users to use the mixture of numbers, letters, and symbols for a stronger password.
+
+#### How do I change my NoteBuddy password?
+
+First, you need to make a backup of your notes.
+Then, click the menu button at the top left of the screen and select **reset NoteBuddy**.
+Note: This process deletes all your notes
```
Comment (YoeriNijs) [inline FAQs .md]: Maybe 'warning' is better than 'note' here.

File: FAQs .md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+A password setup page will appear, where you fill in your new details and click done.
+Click on the menu button at the top left of the screen and select **Import backed up notes**, this would restore your deleted notes back into NoteBuddy.
+A more efficient process would be available in coming updates.
+
+#### I have forgotten my password and can’t log in to NoteBuddy?
```
Comment (YoeriNijs) [inline FAQs .md]: What do I have to do when I have forgotton my password and cannot login anymore? Or something like that.

File: FAQs .md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+In this case, you can only reinstall the app to gain access to NoteBuddy.
+Warning: This process deletes all your saved notes.
+It’s important you back up your notes after each saves. A less painstaking option would be made available in the future.
+
+#### Do I need to pay any fee to use any feature on NoteBuddy?
+
+NoteBuddy is an open source project, and free to use.
+However, you can support the project with a donation or via developing more features. 
+The financial donation is, however, neither compulsory nor obligatory.
```
Comment (YoeriNijs) [inline FAQs .md]: You can skip this. I do not need a financial donation; new pull requests by others, such as your request, are way better ;)

File: FAQs .md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+#### The developer has access to any of my personal information when using the App?
```
Comment (YoeriNijs) [inline FAQs .md]: Has the developer access to any of my personal notes?

File: FAQs .md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+No NoteBuddy do not collect any information when using the app.
```
Comment (YoeriNijs) [inline FAQs .md]: No, NoteBuddy does not collect any user data.

File: FAQs .md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Need to contact the developer?	
+
+Click the menu button at the top left of the screen and select **feedback**.
+
+
+NoteBuddy is an App under development, with more exciting features to be added to enhance user’s experience.
+This document would be updated from time to time, whenever new changes or implementations are made to the software, to help users navigate easily and better with new updates.
+For further inquiries, you can contact the developer via the issue section [here]( https://github.com/YoeriNijs/NoteBuddy/issues)
+
```
Comment (YoeriNijs) [inline FAQs .md]: A lot of whitespaces here; you may remove them.

File: FAQs .md (hunk 1 above)
Comment (omideleayo) [reply FAQs .md]: All noted

File: FAQs .md (hunk 2 above)
Comment (omideleayo) [reply FAQs .md]: rectified

File: FAQs .md (hunk 3 above)
Comment (omideleayo) [reply FAQs .md]: rectified

File: FAQs .md (hunk 13 above)
Comment (omideleayo) [reply FAQs .md]: All done now, check for any more corrections

Comment (omideleayo) [reply FAQs .md]: All done now, check for any more corrections

**Comment by YoeriNijs** [comment]
Merged! Thanks Omideleayo, great work!

**Comment by omideleayo** [comment]
Cool,  it's a pleasure working on nice projects at my leisure

**Pull request merged**